ironfish/src/rpc/routes/wallet/createAccount.ts

```diff
@@ -45,10 +59,23 @@ routes.register<typeof CreateAccountRequestSchema, CreateAccountResponse>(
   CreateAccountRequestSchema,
   async (request, context): Promise<void> => {
     AssertHasRpcContext(request, context, 'wallet')
+    if (
+      request.data.createdAt?.hash === undefined ||
+      request.data.createdAt?.sequence === undefined
+    ) {
+      request.data.createdAt = undefined
```

Comment:
will yup validation enforce both fields being defined this if we add `defined()` to the `hash` and `sequence` fields?